### PR TITLE
Hide registration form if sign up is not enabled

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**
image

### CODE REVIEW

File: projects/arlas-toolkit/assets/i18n/es.json [hunk 1]
```diff
@@ -82,11 +97,10 @@
 	"Send password reset email": "Recibir un correo electrónico de restablecimiento",
 	"Already have an account?": "¿Ya tiene una cuenta?",
 	"Log In": "Iniciar sesión",
-	"interval:": "Intervalo",
-	"x-axis:": "Abscisa :",
-	"From": "Desde",
-	"": "",
-	"y-axis:": "Ordenada :",
+	"x-axis": "",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/es.json:100]: Is it normal they don't have translations ? Is "x-axis:" still in the file ?

File: projects/arlas-toolkit/assets/i18n/en.json [hunk 2]
```diff
@@ -1,300 +1,303 @@
...
-  "An error occured in ARLAS-persistence": "An error occured in ARLAS-persistence.",
-  "accepted": "accepted",
-  "running": "running",
-  "successful": "successful",
-  "failed": "failed",
-  "dissmissed": "dissmissed"
-}
+	"Download": "Download",
+	"Download type": "Download type",
+	"Download archives": "Download archives",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/en.json:4]: missing the

File: projects/arlas-toolkit/assets/i18n/en.json [hunk 3]
```diff
@@ -1,300 +1,303 @@
...
+	"Download archive": "Download archive",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/en.json:5]: missing the

File: projects/arlas-toolkit/assets/i18n/en.json [hunk 4]
```diff
@@ -1,300 +1,303 @@
...
+	"Download images": "Download images",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/en.json:6]: missing the

File: projects/arlas-toolkit/assets/i18n/en.json [hunk 5]
```diff
@@ -1,300 +1,303 @@
...
+	"Download image": "Download image",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/en.json:7]: missing the

File: projects/arlas-toolkit/assets/i18n/en.json [hunk 6]
```diff
@@ -1,300 +1,303 @@
...
+	"No cropping possible": "No cropping possible",
+	"Crop using the geographical filter": "Crop using the geographical filter",
+	"Coordinate system": "Coordinate system",
+	"Coordinate system for the downloaded products": "Coordinate system for the downloaded products",
+	"Format": "Format",
+	"Format of downloaded products": "Format of downloaded products",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/en.json:13]: missing the

File: projects/arlas-toolkit/assets/i18n/en.json [hunk 7]
```diff
@@ -1,300 +1,303 @@
...
+	"Cancel": "Cancel",
+	"Next": "Next",
+	"Parameters": "Parameters",
+	"Summary of parameters": "Summary of parameters",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/en.json:17]: missing the

File: projects/arlas-toolkit/assets/i18n/en.json [hunk 8]
```diff
@@ -1,300 +1,303 @@
...
+	"Validate": "Validate",
+	"native": "Native",
+	"Bookmark already exists": "A bookmark already exists with this name.",
+	"Bookmark creation error": "Error during bookmark creation.",
+	"Bookmark succefully created.": "Bookmark succefully created.",
+	"Unauthorized to create a dashboard, you need to log in": "Unauthorized to create a dashboard, you need to log in",
+	"Missing permissions to create a dashboard": "Missing permissions to create a dashboard",
+	"A configuration with this name exists already, please choose another name": "A dashboard with this name exists already, please choose another name",
+	"An error occurred, please try later": "An error occurred, please try later",
+	"Loading data of": "Loading data of",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/en.json:193]: Shouldn't there be a variable for this string ?

File: projects/arlas-toolkit/src/lib/components/register/register.component.html [hunk 9]
```diff
@@ -4,7 +4,7 @@
       <div class="title">
         <h2>ARLAS - {{'Sign up' | translate}}</h2>
       </div>
-      <div>
+      <div *ngIf="allowRegister; else notAllowed">
         <div class="subtitle">
           {{'It’s quick and easy !' | translate}}
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/src/lib/components/register/register.component.html:9]: There is no space before a ! in english and spanish => to change only in i18n file ?

**Review by QuCMGisaia — APPROVED** [review]

File: projects/arlas-toolkit/src/lib/components/register/register.component.html (hunk 9 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/src/lib/components/register/register.component.html:9]: done

File: projects/arlas-toolkit/assets/i18n/en.json (hunk 8 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/assets/i18n/en.json:193]: I agree but it is not how it is implemented , so i prefer to treat it in another PR

File: projects/arlas-toolkit/assets/i18n/en.json (hunk 7 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/assets/i18n/en.json:17]: done

File: projects/arlas-toolkit/assets/i18n/en.json (hunk 6 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/assets/i18n/en.json:13]: done

File: projects/arlas-toolkit/assets/i18n/en.json (hunk 5 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/assets/i18n/en.json:7]: done

File: projects/arlas-toolkit/assets/i18n/en.json (hunk 2 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/assets/i18n/en.json:4]: done

File: projects/arlas-toolkit/assets/i18n/es.json (hunk 1 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/assets/i18n/es.json:100]: I'd check in another PR, sorry

**Pull request merged**